src/Interpreters/DDLWorker.h

```diff
@@ -1,12 +1,15 @@
 #pragma once
 
+#include <DataStreams/BlockIO.h>
 #include <Interpreters/Cluster.h>
 #include <Interpreters/Context.h>
-#include <DataStreams/BlockIO.h>
+#include <Storages/IStorage.h>
```

Comment:
```suggestion
#include <Storages/IStorage_fwd.h>
```

It does not use anything from the IStorage, it just need the declaration